WordPress/UITestsFoundation/Screens/MySiteScreen.swift

```diff
@@ -1,126 +1,140 @@
 import ScreenObject
 import XCTest
 
-private struct ElementStringIDs {
-    static let navBarTitle = "my-site-navigation-bar"
-    static let blogTable = "Blog Details Table"
-    static let removeSiteButton = "BlogDetailsRemoveSiteCell"
-    static let activityLogButton = "Activity Log Row"
-    static let jetpackScanButton = "Scan Row"
-    static let jetpackBackupButton = "Backup Row"
-    static let postsButton = "Blog Post Row"
-    static let mediaButton = "Media Row"
-    static let statsButton = "Stats Row"
-    static let peopleButton = "People Row"
-    static let settingsButton = "Settings Row"
-    static let domainsButton = "Domains Row"
-    static let createButton = "floatingCreateButton"
-    static let ReaderButton = "Reader"
-    static let switchSiteButton = "SwitchSiteButton"
-    static let dashboardButton = "Home"
-    static let segmentedControlMenuButton = "Menu"
-    // "Free To Paid Plans" Card
-    static let freeToPaidPlansCardId = "dashboard-free-to-paid-plans-card-contentview"
-    static let freeToPaidPlansCardHeaderButton = "Free domain with an annual plan"
-    // "Pages" Card
-    static let pagesCardId = "dashboard-pages-card-frameview"
```

Comment:
I'd actually vote for keeping this, `freeToPaidPlansCardId` and `activityLogCardId`.

On the example with `pagesCardId`, after the changes, the `dashboard-pages-card-frameview` string is used four times in getters, and also used for a `pagesCardId` string. 

WDYT?

Reply:
good eye! yeah looks like quite a bit of repetition there, reverted that in https://github.com/wordpress-mobile/WordPress-iOS/pull/21133/commits/69451c02444646a7760b1e994953d91c5d9ccf69